tool/pylib/generator/config/Manifest.py

```diff
@@ -104,7 +104,8 @@ def patchLibEntry(self, libentry):
     @classmethod
     def schema_v1_0(self):
         patterns = {
-            "semver": r"^trunk$|master$|^\d+\.\d+(\.\d+)?(-[0-9]+-?)?([-a-zA-Z+][-a-zA-Z0-9.:-]*)?$",
+            # semver validation, see https://github.com/sindresorhus/semver-regex/blob/master/index.js
```

Comment:
The `^` and `$` that were in the deleted line are important to match the entire string.

Reply:
Updated the rege accordingly...